modules/bluima_opennlp/src/main/java/ch/epfl/bbp/uima/ae/TokenAnnotator.java

```diff
@@ -55,8 +58,8 @@
      */
     public static final String COMPONENT_ID = "de.julielab.jules.OpenNLPTokenizer";
 
-    @ConfigurationParameter(name = BlueUima.PARAM_MODEL_FILE)
-    private String modelFile;
+    @ConfigurationParameter(name = BlueUima.PARAM_MODEL)
```

Comment:
Ok np, but in the future, avoid to change parameter name if possible, as it might break pipeline scripts that I wrote (there are > 100 pipeline scripts...)